src/axis1.c

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
`axis.c` and propagated down.

Reply:
resolved for now.  address with https://github.com/NSLS-II-CSX/csxtools/issues/99